guides/common/modules/proc_assigning-content-view-environments-to-activation-keys.adoc

```diff
@@ -0,0 +1,24 @@
+[id="assigning-content-view-environments-to-activation-keys"]
+= Assigning content view environments to activation keys
+
+You can assign multiple content view environments to an activation key to provide hosts with access to content from each assigned environment.
+Hosts registered with the activation key receive the environments in the order they are stored.
+For more information, see xref:content-view-environment-ordering-and-priority[Content view environment ordering and priority].
+
+.Prerequisites
+The *Allow multiple content views* setting must be enabled.
+For more information on the *Allow multiple content views* setting, see xref:multiple-activation-keys-and-content-view-environments[].
+
+.CLI procedure
+. Display the available content view environments to obtain their labels.
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer content-view-environment --list
+----
+. Create an activation key and assign multiple content view environments:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer activation-key update --organization _"My_Organization"_ --content-view-environments _"My_env1/My_cv1,My_env2/My_cv2"_
```

Comment:
I think you need to identify the activation key somehow - an option missing, such as AK ID?

Reply:
Yes, as @vijaysawant said here --> https://github.com/theforeman/foreman-documentation/pull/3609#pullrequestreview-2630267866